src/Config.php

```diff
@@ -41,6 +41,11 @@ public static function getReqBodyScrubbers()
         return self::$options['request']['bodyScrubbers'];
     }
 
+    public static function getReqPostFieldScrubbers()
+    {
+        return self::$options['request']['postFieldScrubbers'];
```

Comment:
You're going to need to add `postFieldScrubbers` to `Config::defaultConfig()`, otherwise you'll get an undefined index.